pkg/controllers/managementuser/alert/configsyncer/configsyncer.go

```diff
@@ -255,8 +255,10 @@ func (d *ConfigSyncer) sync() error {
 		logrus.Debug("The config stay the same, will not update the secret")
 	}
 
-	if err := d.syncReceiver(notifiers, cAlertGroupsMap, pAlertGroupsMap); err != nil {
-		return errors.Wrapf(err, "Update Webhook Receiver Config")
+	if webhookReceiverEnabled {
+		if err := d.syncReceiver(notifiers, cAlertGroupsMap, pAlertGroupsMap); err != nil {
```

Comment:
This function is used for syncing configure for webhook-receiver, can we rename it?